src/NumericalAlgorithms/Spectral/Spectral.cpp

```diff
@@ -599,3 +601,22 @@ template const DataVector& Spectral::quadrature_weights<
     Spectral::Basis::FiniteDifference, Spectral::Quadrature::FaceCentered>(
     size_t) noexcept;
 /// \endcond
+
+/// \cond
+template <>
+Spectral::Quadrature
+Options::create_from_yaml<Spectral::Quadrature>::create<void>(
```

Comment:
Add a test... presumably a pair of statements like `CHECK(Spectral::Quadrature::Gauss == TestHelpers::test_creation<Spectral::Quadrature>("Gauss"));`

Are there already tests of `CHECK(get_output(Spectral::Quadrature::Gauss) == "Gauss");` ?